### PR TITLE
fix(checkbox): update story contents and their groups

### PR DESCRIPTION
**Pull request by saty2103**
### What does this implement/fix? Explain your changes.
Current PR fixes followings:
- Update story grouping.
- Fix label on Disabled and Size stories.
- ChoiceList is now used for checkbox group.
- Vertical checkbox group options now has 4px top margin.
- setTimeout removed.
- Error story added.
- Column replaced with className w-25 inside overflow story.
...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/checkbox/__stories__/variants/Error.story.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { action } from '@/utils/action';
+import Checkbox from '../../index';
```
Comment (anuradha9712) [inline core/components/atoms/checkbox/__stories__/variants/Error.story.jsx]: Please import checkbox from '@/index'.

File: core/components/atoms/checkbox/__stories__/Group.story.jsx [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+    { label: 'Patient has language barriers', name: 'problem', value: 'patient has language barriers' },
+    {
+      label: 'Mental and behavioral health conditions',
+      name: 'problem',
+      value: 'mental and behavioral health conditions',
+    },
+  ];
+
+  return (
+    <>
```
Comment (anuradha9712) [inline core/components/atoms/checkbox/__stories__/Group.story.jsx]: React fragment `<></>` is redundant here. Please remove this

**Comment by anuradha9712** [comment]
Please refactor the code for following stories:
- **Overflow Behaviour:** Instead of using `Col` please try to use `w-25` or `w-50` classname.
- **Disabled:** Please try to remove redundant code
- **Size:** Please try to remove redundant code
- **Uncontrolled Checkbox:** Please remove unnecessary div container

File: core/components/atoms/checkbox/__stories__/variants/Error.story.jsx (hunk 1 above)
Comment (saty2103) [reply core/components/atoms/checkbox/__stories__/variants/Error.story.jsx]: done

File: core/components/atoms/checkbox/__stories__/Group.story.jsx (hunk 2 above)
Comment (saty2103) [reply core/components/atoms/checkbox/__stories__/Group.story.jsx]: done

**Comment by anuradha9712** [comment]
Please remove the redundant stories and group the stories properly

**Comment by anuradha9712** [comment]
Please fix the spacing in nested checkbox

**Comment by anuradha9712** [comment]
Check the spacing between checkbox in horizontal alignments

**Comment by anuradha9712** [comment]
In `size` story, user is not able to unchecked the checkbox

File: core/components/atoms/checkbox/__stories__/variants/Size.story.jsx [hunk 3]
```diff
@@ -10,12 +9,11 @@ export const size = () => {
     <div className="d-flex">
       {sizes.map((CheckboxSize, ind) => {
         return (
-          <div key={ind} className="mr-5">
+          <div key={ind} className="mr-7">
+            <Label withInput={true}>{CheckboxSize.charAt(0).toUpperCase() + CheckboxSize.slice(1)}</Label>
             <div className="h-50">
```
Comment (anuradha9712) [inline core/components/atoms/checkbox/__stories__/variants/Size.story.jsx]: Please remove this `h-50` classname

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**